lxd-benchmark/main.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/lxc/lxd/shared/version"
 )
 
-var argCount = gnuflag.Int("count", 100, "Number of containers to create")
+var argCount = gnuflag.Int("count", 3, "Number of containers to create")
```

Comment:
I'd rather we keep this to 100 but change the default image to alpine/edge.

The goal of lxd-benchmark isn't to test how much RAM your containers can use, but to benchmark LXD itself, so switching the default image seems like a better move.

Reply:
While trying `lxd-benchmark` for the first time on a 8GB RAM server, I crashed it using the default count of 100. ZFS was giving errors and the server had to be rebuilt. 
lxd-benchmark would not return and I could not cancel the benchmark.

I had no idea that 100 containers would be too many. I suppose many users do not know that either.

There is the principle of least surprise when trying out software. `ab` (ApacheBench) defaults to a single connection, and in the `man` page it says _The default  is to just perform a single request which usually leads to non-representative benchmarking results._ 
I strongly believe that the default count should be something small, either 3 or even 1.

Reply:
We could switch to alpine by default and use twice the number of cores as the number of containers, this would still give some vaguely useful results without using much resources.